src/cli/lib/printer/printer.ts

```diff
@@ -47,6 +49,19 @@ type Style = {
 	decoration?: Decoration;
 };
 
+const printLoader = () => {
```

Comment:
We don't have a linter rule to force typescript to specify the return value, but just in case you want to practice with it, you can add what the `printLoader` method returns.

`const printLoader = (): number => {`

Reply:
Thanks Iñigo! I'll add it :)